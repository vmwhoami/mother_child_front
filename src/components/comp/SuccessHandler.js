import React from 'react';

import css from '../../css/errorsSuccess.module.css';

const SuccessHandler = () => (
  <div className={css.success}>
    <span>Registration Successfull</span>
  </div>
);

export default SuccessHandler;
