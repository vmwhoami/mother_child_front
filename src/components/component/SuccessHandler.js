import React from 'react';
import PropTypes from 'prop-types';
import css from '../../css/errorsSuccess.module.css';

const SuccessHandler = ({ message }) => (
  <div className={css.success}>
    <span>
      {message}
      {' '}
      Successfull
    </span>
  </div>
);

SuccessHandler.propTypes = {
  message: PropTypes.string.isRequired,
};
export default SuccessHandler;
