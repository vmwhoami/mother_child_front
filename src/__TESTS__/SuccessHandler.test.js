import React from 'react';
import renderer from 'react-test-renderer';
import SuccessHandler from '../components/component/SuccessHandler';

const message = 'Success';

it('renders SuccessHandler correctly', () => {
  const tree = renderer.create(<SuccessHandler message={message} />);
  expect(tree).toMatchSnapshot();
});
