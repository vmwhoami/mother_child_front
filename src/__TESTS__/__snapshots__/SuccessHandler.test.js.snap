// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders SuccessHandler correctly 1`] = `
<div
  className="success"
>
  <span>
    Success
     
    Successfull
  </span>
</div>
`;
